src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -111,10 +115,15 @@ Copyright (c) .NET Foundation. All rights reserved.
                             '$(EnableCompressionInSingleFile)' == 'true' And
                             '$(SelfContained)' != 'true'"
                  ResourceName="CompressionInSingleFileRequiresSelfContained" />
+    <NETSdkError Condition="'$(PublishAot)' == 'true' And
+                            '$(_SkipAotSupportedRidCheck)' != 'true' And
```

Comment:
We have similar property in 
https://github.com/dotnet/runtime/blob/cb6fb60dfc253bc13655fafa15686cb864fd0ea2/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets#L58 . 

Would it make sense to use one property to suppress all these supported checks?

- Rename `_SkipAotSupportedRidCheck` -> `SkipAotSupportedCheck` here
- Rename `DisableUnsupportedError` to `SkipAotSupportedCheck` in dotnet/runtime.

Reply:
That makes sense, but I'd like to do that as a separate change, as I intend for this to go into .NET 7 as a servicing fix.

Reply:
If you agree that we should use the same property name in all places, the property should be renamed in this servicing fix.